src/components/Input/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Form } from 'react-bootstrap';
+
+interface IInputProps {
+    label: string;
+    type: string;
+    name: string;
+    formik: Record<string, any>;
+    isBlur: boolean;
+    isError: boolean;
+}
+
+export const Input = ({
+    label,
+    name,
+    formik,
+    isBlur = false,
+    isError = false,
+    type = 'text',
+}: IInputProps) => (
+    <Form.Group className="mb-3">
+        <Form.Label>{label}</Form.Label>
+        <Form.Control
+            type={type}
+            name={name}
+            onChange={formik.handleChange}
```

Comment:
В базовых компонентах лучше иметь минимум зависимостей. Тогда их станет проще переиспользовать в разных частях приложения.
Тут лучше убрать зависимость от `formik` и все значения принимать через пропсы

```suggestion
            onChange={onChange}
```